app/src/main/res/values/strings.xml

```diff
@@ -69,10 +69,7 @@
     <item quantity="other">%d uploads</item>
   </plurals>
   <string name="categories_not_found">No categories matching %1$s found</string>
-  <string name="categories_skip_explanation">Add categories to make your images more discoverable on Wikimedia Commons.
-
-Start typing to add categories.
```

Comment:
I don't think this should have been removed?

Reply:
You're right, i'm fixing it now :)